lib/trailer_vote/fixtures/application/vnd.trailervote.push_campaign_trivial.v4+json

```diff
@@ -0,0 +1,28 @@
+{
+  "push_campaign_trivial": {
+    "_links": {
+      "self": {
+        "href": "https://fake.dev.trailervote.com/trailervote-api/admin/push_trivial_campaigns/9664ef93-5c38-4409-9652-f0c51c61377a"
+      },
+      "action": {
+        "href": "http://cinema1.dev.trailervote.com/venom.php"
+      },
+      "rich_media": {
+        "href": "https://link.some.com/image.png"
+      }
```

Comment:
Can this also be audio? If so, is it possible to combine with an image?

Reply:
Afaik, we can only add images to rich notifications. Confirmed by Danil